lib/Frontend/nvqpp/ConvertExpr.cpp

```diff
@@ -1574,6 +1514,76 @@ bool QuakeBridgeVisitor::VisitCallExpr(clang::CallExpr *x) {
             }
       }
 
+    if (isInNamespace(func, "driver")) {
```

Comment:
I think from a language specification perspective, these in-kernel malloc/malloc_set/free functions are not quite ready for prime time. Still a bit to work out here.

Reply:
Removed in afae659d